packages/mobile/src/components/icons/icons.tsx

```diff
@@ -66,6 +67,18 @@ const LogInIcon: AppIcon = (iconProps) => (
   <FeatherIcon name="log-in" {...iconProps} />
 );
 
+const PlusIcon: AppIcon = (iconProps) => (
+  <FeatherIcon name="plus-circle" {...iconProps} />
+);
+
+const ImageIcon: AppIcon = (iconProps) => (
+  <FeatherIcon name="image" {...iconProps} />
+);
+
+const InfoIcon: AppIcon = (iconProps) => (
+  <Icon name="info-circle" {...iconProps} />
+);
```

Comment:
Let's use one icon set as long as it has required icons. I mean we may use a bit different-looking icon with the same meaning from the existing set